src/base/solver.jl

```diff
@@ -80,11 +80,11 @@ not be changed.
 function set_optmodel_bound!(
     vidx,
     opt_model;
-    lb::Maybe{Real} = nothing,
-    ub::Maybe{Real} = nothing,
+    lower::Maybe{Real} = nothing,
```

Comment:
Same `_bound` comment

Reply:
Maybe check that I did not miss a `lower` or `upper` somewhere

Reply:
checked with grep